tools/Makefile.unix

```diff
@@ -697,3 +704,8 @@ apps_distclean:
 ifneq ($(APPDIR),)
 	$(Q) $(MAKE) -C "$(TOPDIR)/$(APPDIR)" TOPDIR="$(TOPDIR)" distclean
 endif
+
+check_format:
+	$(call colorecho,'Checking NuttX formatting with nxstyle')
```

Comment:
Description of all new make targets should be included in the top-level Makefile, including a description of what the make target does.

Reply:
```
# This is a top-level "kludge" Makefile that just includes the correct
# Makefile.  If you already know the Makefile that you want, you can skip
# this nonsense using:
#
#   make -f Makefile.unix, OR
#   make -f Makefile.win
#

-include .config
ifeq ($(CONFIG_WINDOWS_NATIVE),y)
include tools/Makefile.win
else
include tools/Makefile.unix
endif
```

@patacongo Here?

Reply:
No, I meant README not Makefile.  Here:

`1354 Build Targets and Options`

Reply:
Ah! Ok Wild do